homeassistant/components/wolflink/sensor.py

```diff
@@ -79,6 +105,16 @@ def extra_state_attributes(self):
             "parent": self.wolf_object.parent,
         }
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, self._device_id)},
+            name=self._device_name,
```

Comment:
Not necessary, the title of the config entry is used by default, which, if I've seen correctly, is the device name. The passing of `_device_name` is therefore not necessary at all.